frontends/web/src/components/message/message.tsx

```diff
@@ -14,21 +14,21 @@
  * limitations under the License.
  */
 
-import { h, RenderableProps } from 'preact';
-import * as styles from './message.module.css';
+import React, { PropsWithChildren } from 'react';
+import styles from './message.module.css';
 
 export interface Props {
     hidden?: boolean;
     type?: 'message' | 'success' | 'info' | 'warning' | 'error';
-    style?: string;
+    style?: React.CSSProperties;
 }
 
 export function Message({
     hidden,
     type = 'message',
-    style = '',
+    style,
```

Comment:
style is not used anymore in the codebase, except in a test, we could remove or pass with `...props`

Reply:
I made an issue #1535